test/unit-tests/function/signal/zpk2tf.test.js

```diff
@@ -0,0 +1,26 @@
+import approx from '../../../../tools/approx.js'
+import math from '../../../../src/defaultInstance.js'
+
+const zpk2tf = math.zpk2tf
+
+describe('zpk2tf', function () {
+  it('should return the transfer function of a zero-pole-gain model', function () {
+    approx.deepEqual(zpk2tf([math.complex(1, 5)], [math.complex(-2, 0), math.complex(-3, 0)], 1), [[math.complex(1, 0), math.complex(-1, -5)], [math.complex(1, 0), math.complex(5, 0), math.complex(6, 0)]])
```

Comment:
😂 how wide a monitor do you use?  These tests have a line length up to 335 characters. Can you reformat this so it is max 100 characters or so? That will make it better readable.

Reply:
I actually have a wide monitor I didn't notice it was too long 😂. I reformatted it.